### PR TITLE
Update differences doc

### PR DESCRIPTION
**Pull request by knubez**

### CODE REVIEW

**Comment by zmoon** [comment]
The CI failure is because the env got a too-low regionmask version (0.4.0, early 2018). `.from_geopandas` was added in [v0.6.0, mid 2020](https://regionmask.readthedocs.io/en/stable/whats_new.html#v0-6-0-30-07-2020).

**Comment by zmoon** [comment]
Not sure why that test is failing now.

<details><summary>Env differences from passing -> failing</summary>

| name             | version_old   | version_new   |
|:-----------------|:--------------|:--------------|
| asttokens        | 2.0.5         | 2.0.8         |
| executing        | 0.9.1         | 0.10.0        |
| giflib           | 5.2.1         | -             |
| libsqlite        | -             | 3.39.2        |
| libwebp          | 1.2.3         | -             |
| libwebp-base     | 1.2.3         | 1.2.4         |
| nbconvert        | 6.5.2         | 6.5.3         |
| nbconvert-core   | 6.5.2         | 6.5.3         |
| nbconvert-pandoc | 6.5.2         | 6.5.3         |
| openjpeg         | 2.4.0         | 2.5.0         |
| partd            | 1.2.0         | 1.3.0         |
| pytz             | 2022.1        | 2022.2.1      |
| pyzmq            | 23.2.0        | 23.2.1        |
| setuptools       | 63.4.2        | 65.0.0        |
| stack_data       | 0.3.0         | 0.4.0         |
| xz               | 5.2.5         | 5.2.6         |
</details>

**Comment by fsenf** [comment]
Thank you for being interested in my opinion. As far as I can tell, none of the statements regarding tobac are wrong. So it fits and represents the current state of development quite well.

In any case, I suggest that in the future we should also focus on the commonalities of our tracking approaches. Maybe this will result in something special.

File: docs/differences.md [hunk 1]
```diff
@@ -1,15 +1,40 @@
...
+  Note that the major version of the Python package
+  is not (currently) consistent with the "v2.0".
+  Currently, the major version is 0, to indicate that the API is not stable yet.
+<!-- prettier-ignore-end -->
+
+## TAMS vs [tobac](https://tobac.readthedocs.io)
+
+- tobac[^tob] is a more general cloud tracking toolkit with more options, etc., while TAMS targets the MCS case (though much of the core API is purposely left more general)
+- tobac treats features as single points, while TAMS treats them as georeferenced polygonal areas (Shapely)
+- tobac separates feature identification and segmentation, such that the geo areas of the original identified features based on thresholds can be different than the segmentation areas, which are calculated with a watershed method. In TAMS, the feature geo area is treated as the segmentation area.
```
[the remark this answers is not in the document]
Comment (zmoon) [reply docs/differences.md]: 7b756fe

**Pull request merged**